contracts/Nexus.sol

```diff
@@ -102,8 +102,8 @@ contract Nexus is INexus, BaseAccount, ExecutionHelper, ModuleManager, UUPSUpgra
     {
         address validator = op.nonce.getValidator();
         if (!op.nonce.isModuleEnableMode()) {
-            // Check if validator is not enabled. If not, return VALIDATION_FAILED.
-            if (!_isValidatorInstalled(validator)) return VALIDATION_FAILED;
+            // Check if validator is not enabled. If not, revert.
+            require(_isValidatorInstalled(validator), InvalidModule(validator));
```

Comment:
> InvalidModule

shouldn't this error be more readable and specific?
**ValidatorNotInstalled**

Reply:
This is tricky, if nonce is incorrect it will fail with InvalidModule or ValidatorNotInstalled as @livingrockrises recommended, anyway it will confuse the user. We could add a check and revert with InvalidNonce but most of the times the nonce will be correct so not worth adding this check.

Reply:
I would revert with a "ValidatorNotInstalled" error, you are checking if validator is installed, if not you will revert with a custom error so it is more clear to name it something like "ValidatorNotInstalled" imo